src/codegeneration/ModuleTransformer.js

```diff
@@ -87,7 +87,7 @@ export class ModuleTransformer extends TempVarTransformer {
   }
 
   getTempVarNameForModuleSpecifier(moduleSpecifier) {
-    let normalizedName = System.normalize(moduleSpecifier.token.processedValue, this.moduleName);
+    let normalizedName = $traceurRuntime.normalizeModuleName(moduleSpecifier.token.processedValue, this.moduleName);
```

Comment:
We are only using normalize to place the specifier in a namespace that will make the resulting tempvar unique.  We can apply a local function instead.